portfolio_app/serializers.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import serializers
+from .models import Portfolio
+from property_app.serializers import PropertySerializer
+
+class PortfolioSerializer(serializers.ModelSerializer):
+
+    properties = PropertySerializer(many=True)
+
+    class Meta:
+        model = Portfolio
+        fields = ['id', 'name', 'properties']
+
+    def get_properties(self, obj):
```

Comment:
What does `get_properties` do in this case. It looks like it's a method serializer for some form of association but it is never being called or used.